### DISCUSSION TITLE
Queue Command

### DISCUSSION THREAD

**Discussion by LoganHeasman200** [TannerGabriel/discord-bot#90] [Ideas]
Would it be possible to add a command to view the queue of music that is currently playing and will be playing next, I'm not quite sure how the queue is stored and how to access it.

**Comment by TannerGabriel** [comment]
@LoganHeasman200 Yes, this is possible. Basically, you would just need to print the songs from the queue object. I will probably be able to implement the feature this weekend. You could also try implementing it yourself and make a PR to the repo if you would like.

**Reply by TannerGabriel** [reply]
For reference, this feature has been implemented #91.
